tests/models/depth_anything/test_modeling_depth_anything.py

```diff
@@ -236,26 +236,24 @@ def _validate_backbone_init():
 
 
 # We will verify our results on an image of cute cats
-def prepare_img():
-    image = Image.open("./tests/fixtures/tests_samples/COCO/000000039769.png")
-    return image
+def predict_depth(image_processor, model):
+    image = image = Image.open("./tests/fixtures/tests_samples/COCO/000000039769.png")
+    inputs = image_processor(images=image, return_tensors="pt").to(torch_device)
+    with torch.no_grad():
+        outputs = model(**inputs)
+    return outputs.predicted_depth
```

Comment:
Let's move the predict logic back into the test and keep with `prepare_img` -- this is more consistent with other tests and doesn't abstract away the important part: it's better for tests to be clear and explicit